contract/src/utils.rs

```diff
@@ -625,8 +625,7 @@ pub(crate) fn migrate_token_hashes(token_owner: Key) {
     storage::write(unmatched_hash_count_uref, unmatched_hash_count);
 }
 
-// This function is incredibly gas expensive
-// DO not use this function unless absolutely necessary.
+
```

Comment:
This additional line does not pass linting

Reply:
Ran make lint